controllers/secretdefinition_controller_test.go

```diff
@@ -0,0 +1,192 @@
+package controllers
+
+import (
+	"context"
+	"encoding/base64"
+
+	//"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	secretsmanagerv1alpha1 "github.com/tuenti/secrets-manager/api/v1alpha1"
+	"github.com/tuenti/secrets-manager/errors"
+
+	//corev1 "k8s.io/api/core/v1"
+	"reflect"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	encodedValue = "bG9yZW0gaXBzdW0gZG9ybWEK"
+	decodedValue = "lorem ipsum dorma"
+)
+
+var _ = Describe("SecretsManager", func() {
```

Comment:
(curious) is there any need to store the "description" in a var?

Reply:
test suite is just generated code, that you fill up with test cases.